storage/backend/azure/azure.go

```diff
@@ -46,8 +46,13 @@ func New(l log.Logger, c Config) (*Backend, error) {
 	// 3. Azurite has different URL pattern than production Azure Blob Storage.
 	var blobURL *url.URL
 	if c.Azurite {
+		fmt.Println("Container Name: %s", c.ContainerName)
+		level.Info(l).Log("Container name: ", c.ContainerName)
 		blobURL, err = url.Parse(fmt.Sprintf("http://%s/%s/%s", c.BlobStorageURL, c.AccountName, c.ContainerName))
 	} else {
+		// add print statement
+		fmt.Println("Container Name: %s", c.ContainerName)
```

Comment:
same here

Reply:
I have made the changes. Thank you for noticing!